Components/Avatar/avatar.css

```diff
@@ -0,0 +1,55 @@
+body{
+    font-family: 'Quicksand', sans-serif;
+    font-size: 20px;
+}
+
+.avatar-container{
+    border-radius: 1rem;
+    box-shadow:inset 0rem 0rem 0.5rem gray;
+    padding: 1rem;
+    margin: 0.5rem;
+    max-width: 100vw;
+}
+
+h2{
+    margin: 0rem 1rem;
+}
+
+.avatar-container ul{
+    text-align: center;
+    display: block;
+    padding: 0rem;
+    margin: 0rem;
+}
+
+.avatar-container li{
+    display: inline;
+}
+
+.avatar-biggest{
+    border-radius: 50%;
+    width: 7rem;
+    height: 7rem;
+    margin: 0.5rem 1rem;
+}
+
+.avatar-big{
+    border-radius: 50%;
+    width: 5rem;
+    height: 5rem;
+    margin: 0.5rem 1rem;
+}
+
+.avatar-small{
+    border-radius: 50%;
+    width: 3rem;
+    height: 3rem;
+    margin: 0.5rem 1rem;
+}
+
+.avatar-smallest{
+    border-radius: 50%;
+    width: 2rem;
+    height: 2rem;
+    margin: 0.5rem 1rem;
+}
```

Comment:
Suggestions:
- You can make a common class for border-radius and margin to get rid of repetitions like this
`.avatar {
    border-radius: 50%;
    margin: 0.5rem 1rem;
}`
`.avatar-small {
    define only width and height here
}`